shap/plots/_image.py

```diff
@@ -190,6 +190,8 @@ def image_to_text(shap_values):
         for each sample
 
     """
+    if not have_ipython:
+        raise ImportError("IPython is required for this function but could not be imported")
```

Comment:
```suggestion
        raise ImportError("IPython is required for this function but could not be imported. Run `pip install ipython` to fix this.")
```

This last part you added in `_text.py` is more informative and gives direction to the user.

Reply:
Good idea